protocol/database-scripting/README.md

```diff
@@ -0,0 +1,102 @@
+perform a COPY of the data from the Temp Table into the original table (and a deletion
+of updated records in the case of an Update).
+
+Code Review
+-----------
+
+Typically, console scripts are not code that we want to add to the Healthify app's
+code repo. As a result, to solicit Code Review you can put your code in a Github
+[gist](https://help.github.com/articles/about-gists/) and share the gist's link
+with your teammates to get Code Review. Also, be sure to include a link to the gist
+in the description of the corresponding Pivotal Story.
+
+Running database scripts poses a higher risk than deploying a normal feature. Hence,
+it's important to have a higher bar of Code Review than for normal stories. Make sure
+you solicit Code Review from at least two independent reviewers.
+
+Pre-execution + QA
+-------------------
+
+Before executing the script, generate a backup of the database you will be running the script
+on. This can be done with the Aptible CLI. E.g.
+
+```
+  aptible db:backup production-healthify-postgresql
+```
+
+You can view database backups in the Aptible Dashboard by viewing the show page of
+a particular database and clicking the "Backups" tab. Take note of which backup
+you just created (the backups are timestamped). In the event that there is a problem,
+the backup can be used as the source of truth about what the affected database *should*
+look like.
+
+To QA the script, perform the exact the same workflow on staging as you intend for production.
+1) Create a backup of the staging database.
+2) Execute the script, confirming that it first performs the steps to ensure
+  reversibility (e.g. creates temp tables of the necessary data).
+3) Sign in to the app and explore its functionality, validating that no data was botched.
+
+Having done this QA, now confirm that you can successfully reverse the script if
+necessary. Execute your remediation strategy as planned for during development, and explore
+the app to confirm that you were able to successfully undo the changes. Once this is
+confirmed, feel free to re-run the script if that would result in a desirable state for
+staging.
+
+As always (but particularly for database scripts), make sure the QA-er is distinct
+from the developer.
+
+Deployment
+----------
+
+Having successfully QA-ed the script on Staging, we can move to execute it on Production.
+
+Prior to execution, make sure to alert the dev team and relevant members of the Leadership,
+Customer Success, and/or Resource Network teams of the impending changes. Give them time
+to provide any warnings or other feedback before you execute the script.
+
+Assuming approval from them, perform the same steps from QA:
+1) Create a backup of the staging database.
+2) Execute the script, confirming that it first performs the steps to ensure
+  reversibility (e.g. creates temp tables of the necessary data).
+3) Sign in to the app and explore its functionality, validating that no data was botched.
```

Comment:
Should this also be someone who's not the developer, or at least someone who is pairing with the developer?